moncli/entities/column_value/simple.py

```diff
@@ -117,7 +117,46 @@ class PhoneValue(ComplexNullValue):
 
 class StatusValue(ComplexNullValue):
     """A status column value."""
+    native_type = str
+    allow_casts = (int, str)
 
+    def _convert(self, value):
+        settings = self.settings
+        labels = settings['labels']
+        index = str(value['index'])
+        value = labels[index]  
+        return value
+    
+    def _cast(self, value):
+        if isinstance(value,int):
+            index = str(value)
+            labels = self.settings['labels']
+            try:
+                value = labels[index]
+                return value
+            except KeyError:
+                raise ColumnValueError( 'invalid_status_index',self.id )
+        if isinstance(value,str):
+            labels = self.settings['labels']
+            try:
+                int_value = int(value)
```

Comment:
Funny that you did it this way.  As we are only trying to trigger the exception in the case where the string value is not an int and could be a label, you do not need to assign the result to a variable that will not be used.  The code works fine with it there, but it is cleaner without the variable assignment.